src/css/typography.sass

```diff
@@ -66,3 +66,20 @@ h6
 .text-caption
   font-size: 1.2rem
   font-family: "ralewayitalic"
+
+.wallet-btn
```

Comment:
- Just on general note: A component-specific css class like wallet-btn would be better placed in the app.sass or a new sass file than in "typography".
- The font-family is ralewayregular by default, so we can drop that property.
- The border-radius is something that I would like to have the same for all buttons, so let's specify that in app.sass under `.q-btn`
- Nice work with the grey border style. I like it! It might be worth while creating a special class just for this type of button. Let's call it `.outline-btn` and place it in app.sass. Then we can remove this `wallet-btn` class and references to it altogether :upside_down_face: